src/styles/header.scss

```diff
@@ -0,0 +1,113 @@
+header {
```

Comment:
Could we add a className ".header" instead of just tag? It could prevent some style conflicts with other components that also could have tag header.